posthog/hogql_queries/web_analytics/web_overview.py

```diff
@@ -177,16 +191,31 @@ def previous_period_aggregate(function_name, column_name, alias, params=None):
                 self.query_compare_to_date_range.date_to_as_hogql(),
                 alias=alias,
                 params=params,
+                extra_args=extra_args,
             )
 
+        session_id_is_not_null = ast.Call(name="isNotNull", args=[ast.Field(chain=["session_id"])])
```

Comment:
Nice, tbh in future I can see us only adding this if any session properties are set in the query. Out of scope for this PR though, I'm just rambling about ideas. It would solve back end conversion events

Reply:
Yeah, true. This is my attempt to differentiate between events and dw rows, but I could differentiate them in a different way if we ever drop the `session_id` requirement